lib/ebae_web/controllers/registration_controller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule EbaeWeb.RegistrationController do
+  use EbaeWeb, :controller
+
+  alias Ebae.{Accounts, Accounts.User, Accounts.Guardian}
+
+  def new(conn, _) do
+    if authenticated?(conn) do
+      conn
+      |> put_flash(:info, "Already signed in")
+      |> redirect(to: "/")
+    else
+      render(conn, "new.html",
+        changeset: user_changeset(),
+        action: Routes.registration_path(conn, :create)
+      )
+    end
+  end
+
+  defp authenticated?(conn) do
```

Comment:
This is dup'd between controllers -> extracted out into module?

Reply:
Extracted all the required Guardian methods into an Auth module that lives in the EbaeWeb domain. Seems DRYer and more consistent with my testing pattern.